rosidl_typesupport_connext_c/resource/msg__type_support_c.cpp.em

```diff
@@ -322,6 +322,20 @@ else:
   return true;
 }
 
+static bool get_serialized_length(unsigned int * expected_length)
```

Comment:
I know the other functions are structured this way, but I think we should start to move towards the ability to set and report error states. So instead of returning a `bool`, I think it would be better to return an `rmw_ret_t` and instead of printing to `stderr` with `fprintf`, I'd like to see us using the error state with `RMW_SET_ERROR_MSG`.

We don't need to refactor all of the generated code here to be the same, but in this instance I'd slightly prefer a trend toward better error reporting over consistency with the rest of the existing file.